common/src/main/scala/codacy/utils/FileHelpers.scala

```diff
@@ -4,31 +4,31 @@ import java.net.{URI, URL, URLClassLoader}
 
 import better.files._
 
-import scala.util.{Failure, Success, Try}
-
 import scala.collection.JavaConversions._
 
 object FileHelpers {
 
-  implicit class UrlExtension(val value:URL) extends AnyVal{
+  implicit class UrlExtension(val value: URL) extends AnyVal {
     def toFile = fileFromUri(value.toURI)
   }
 
-  private def fileFromUri(uri:URI):File = {
-    Try(File(uri)) match{
-      case Success(file) => file
-      case Failure(e:java.nio.file.FileSystemNotFoundException) =>
-        val env = new java.util.HashMap[String,String]
+  private def fileFromUri(uri: URI): File = {
+    def createFileSystemIfNotExists(): Unit = {
+      try {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Default case not specified.](https://app.codacy.com/gh/codacy/codacy-scalameta/pullRequest?prid=5195080)